OWNERS_ALIASES

```diff
@@ -114,8 +113,8 @@ aliases:
   - vaikas
   knative-release-leads:
   - RichieEscarez
-  - lionelvillard
-  - yanweiguo
+  - ZhiminXiang
```

Comment:
Just curious on why / if we still need @RichieEscarez here as a release lead. My understanding was that it was earlier required for Hugo related tasks, but release leads have been able to do this for a few releases now. See below also.